components/list/demo/list-drag-n-drop-sample.js

```diff
@@ -0,0 +1,78 @@
+import '../list-item-content.js';
+import './list-item-sample.js';
+import '../list.js';
+import { html, LitElement } from 'lit-element/lit-element.js';
+import { repeat } from 'lit-html/directives/repeat';
+
+class listDragNDropSample extends LitElement {
```

Comment:
Nit: I think in the past we've done something like `CustomListItem` or `DemoListItem`.

Reply:
That makes sense. I'll fix that.

Reply:
Side note this is actually, a component that uses the list component. But I will name it with Demo in front.